dart/dynamics/detail/MultiDofJoint.h

```diff
@@ -37,6 +37,19 @@
 #ifndef DART_DYNAMICS_DETAIL_MULTIDOFJOINT_H_
 #define DART_DYNAMICS_DETAIL_MULTIDOFJOINT_H_
 
+#define MULTIDOFJOINT_REPORT_DIM_MISMATCH( func, arg )              \
+  dterr << "[MultiDofJoint::" #func "] mismatch beteween size of "  \
+        << #arg " [" << arg .size() << "] and the number of "       \
+        << "DOFs [" << getNumDofs() << "] for Joint named ["        \
+        << getName() << "]\n";                                      \
```

Comment:
I guess if we enforce the use of complete sentences for warning messages (which would probably be good practice), then it would make sense to enforce ending those sentences with periods consistently. I'm on board.